apps/condo/pages/playground/iframe.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
can we please order imports for better readability&

Reply:
this page will be removed before merge 😅